packages/react-data-grid/src/masks/InteractionMasks.tsx

```diff
@@ -692,12 +664,7 @@ export default class InteractionMasks<R, K extends keyof R> extends React.Compon
         onKeyDown={this.onKeyDown}
         onFocus={this.onFocus}
       >
-        {copiedPosition && (
-          <CopyMask
-            {...this.getSelectedDimensions(copiedPosition)}
-            ref={this.copyMask}
-          />
-        )}
+        {copiedPosition && <CopyMask {...this.getSelectedDimensions(copiedPosition)} />}
```

Comment:
what is the `CopyMask`? Like `ctrl c` / `ctrl v` background?

Reply:
![image](https://user-images.githubusercontent.com/567105/66955156-aacfa000-f059-11e9-946c-baa85931531d.png)
A mask that appears when you copy a cell.